EntityManager/src/test/java/eu/neclab/ngsildbroker/entityhandler/controller/EntityBatchControllerTest.java

```diff
@@ -0,0 +1,334 @@
+					  "         \"type\":\"Property\",\r\n" + "         \"value\":81\r\n" +
+					  "    }\r\n" + "},\r\n" + "{  \r\n" +
+					  "   \"id\":\"urn:ngsi-ld:Vehicle:A103\",\r\n" +
+					  "   \"type\":\"Vehicle\",\r\n" + "   \"brandName\":\r\n" + "      {  \r\n" +
+					  "         \"type\":\"Property\",\r\n" + "         \"value\":\"Mercedes\"\r\n"
+					  + "      },\r\n" + "   \"speed\":{  \r\n" +
+					  "         \"type\":\"Property\",\r\n" + "         \"value\":82\r\n" +
+					  "    }\r\n" + "}\r\n" + "]";
+			  
+			  badRequestPayload1 = "[  \r\n" + "{  \r\n" +
+					  "   \"id\":\"urn:ngsi-ld:Vehicle:A101\",\r\n" +
+					  "   \"type123\":\"Vehicle\",\r\n" + "   \"brandName\":\r\n" + "      {  \r\n" +
+					  "         \"type\":\"Property\",\r\n" + "         \"value\":\"Mercedes\"\r\n"
+					  + "      },\r\n" + "   \"speed\":{  \r\n" +
+					  "         \"type\":\"Property\",\r\n" + "         \"value\":80\r\n" +
+					  "    }\r\n" + "},\r\n" + "{  \r\n" +
+					  "   \"id\":\"urn:ngsi-ld:Vehicle:A102\",\r\n" +
+					  "   \"type123\":\"Vehicle\",\r\n" + "   \"brandName\":\r\n" + "      {  \r\n" +
+					  "         \"type\":\"Property\",\r\n" + "         \"value\":\"Mercedes\"\r\n"
+					  + "      },\r\n" + "   \"speed\":{  \r\n" +
+					  "         \"type\":\"Property\",\r\n" + "         \"value\":81\r\n" +
+					  "    }\r\n" + "},\r\n" + "{  \r\n" +
+					  "   \"id\":\"urn:ngsi-ld:Vehicle:A103\",\r\n" +
+					  "   \"type123\":\"Vehicle\",\r\n" + "   \"brandName\":\r\n" + "      {  \r\n" +
+					  "         \"type\":\"Property\",\r\n" + "         \"value\":\"Mercedes\"\r\n"
+					  + "      },\r\n" + "   \"speed\":{  \r\n" +
+					  "         \"type\":\"Property\",\r\n" + "         \"value\":82\r\n" +
+					  "    }\r\n" + "}\r\n" + "]";
+			  
+			  deletePayload = "[  \r\n" + "\"urn:ngsi-ld:Vehicle:A101\",\r\n" +
+							    "\"urn:ngsi-ld:Vehicle:A102\"\r\n" + "]"; 
+  
+
+			  BadRequestDeletePayload = "[  \r\n" + "\"urn:ngsi-ld:Vehicle:A201\",\r\n" +
+					    "\"urn:ngsi-ld:Vehicle:A102\"\r\n" + "]"; 
+			  
+			  //@formatter:on 
+
+	}
+
+	@AfterEach
+	public void tearDown() {
+		payload = null;
+		badRequestPayload = null;
+		badRequestPayload1 = null;
+		deletePayload = null;
+		BadRequestDeletePayload = null;
+
+	}
+
+	/**
+	 * this method is use for create the multiple entity
+	 */
+	@Test
+	@Order(1)
+	public void createMultipleEntityTest() {
+		
+		try {    
+	        ExtractableResponse<Response> response = RestAssured.given()
+	                  .body(payload)
+	                .header(HttpHeaders.CONTENT_TYPE, AppConstants.NGB_APPLICATION_JSON)
+	                 .header(HttpHeaders.ACCEPT,AppConstants.NGB_APPLICATION_JSONLD)
+	                .when()
+	                .post("/ngsi-ld/v1/entityOperations/create")
+	                .then()
+	                   .statusCode(Status.CREATED.getStatusCode())
+	                   .statusCode(201).extract();
+	            int statusCode = response.statusCode();
```

Comment:
No need to declare this variable. you can compare response code directly.

Reply:
review incorporated